src/tools/spark-sstfile-generator/src/main/scala/com/vesoft/nebula/tools/generator/v2/SparkClientGenerator.scala

```diff
@@ -310,7 +309,11 @@ object SparkClientGenerator {
               val values = (for {
                 property <- valueProperties if property.trim.length != 0
               } yield extraValue(row, property)).mkString(",")
-              (String.valueOf(extraValue(row, vertex)), values)
+              (if (String.valueOf(extraValue(row, vertex)).startsWith("\"") && String.valueOf(extraValue(row, vertex)).endsWith("\"")) {
+                String.valueOf(extraValue(row, vertex)).slice(1,-1)
+              } else {
```

Comment:
By the way, you should extraValue only once

Reply:
New pr refers to [https://github.com/vesoft-inc/nebula/pull/2250](#2250)